static/app/views/preprod/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Feature from 'sentry/components/acl/feature';
+import {Alert} from 'sentry/components/core/alert';
+import * as Layout from 'sentry/components/layouts/thirds';
+import NoProjectMessage from 'sentry/components/noProjectMessage';
+import {t} from 'sentry/locale';
+import useOrganization from 'sentry/utils/useOrganization';
+
+type Props = {
+  children: React.ReactNode;
```

Comment:
Nit: I'm guessing this is wrong in many places around Sentry, but React.ReactNode is a nullable type, and this is one of the cases where you probably want `NonNullable<React.ReactNode>` as views should always have something to render